src/screens/signup/index.tsx

```diff
@@ -11,8 +12,10 @@ interface SignupScreenProp extends NavigationInterface {
 export default function SignupScreen(props: SignupScreenProp) {
   return (
     <Container>
-      <Button title="Sign up screen button" />
-      <Welcome>Signup Screen</Welcome>
+      <SafeAreaView>
```

Comment:
SafeAreaView usage is meant to be the top container of the page, that's wrapping your container component,
And don't use react-native safe area view rather use the custom made safe-area-view that works for all platforms and phone sizes, other than the react-native safe-area-view that only works with IOS and not android and Andriod notch, 
This custom component can be seen inside the commons folder